src/lightning_app/cli/lightning_cli.py

```diff
@@ -26,14 +34,23 @@ def get_app_url(runtime_type: RuntimeType, *args) -> str:
         return "http://127.0.0.1:7501/admin" if LOCAL_LAUNCH_ADMIN_VIEW else "http://127.0.0.1:7501/view"
 
 
+def main():
+    if len(sys.argv) == 1:
+        _main()
+    elif sys.argv[1] in _main.commands.keys():
```

Comment:
This means that apps cannot have any commands of lightning such as `app`, `init`, `login` ..... 

Also if lightning CLI decides to use some commands in the future that were used by any of the apps, these apps will break.

Reply:
Yes, this is a limitation but it is fine for now.